suricata.yaml.in

```diff
@@ -977,6 +977,7 @@ app-layer:
 
            # LZMA decompression memory limit.
            #lzma-memlimit: 1 Mb
+           #lzma-enabled: no
```

Comment:
The code enables lzma by default, so this value should be yes, but still commented out to reflect the default.

Reply:
Ok, I did not know that convention